src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,78 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { filterTodo } from '../../features/filterTodo';
+import { actions } from '../../features/currentTodo';
 
 export const TodoModal: React.FC = () => {
+  const [currentUser, setCurrentUsers] = useState<User>();
```

Comment:
Consider renaming 'setCurrentUsers' to 'setCurrentUser' since it sets a single user.

Reply:
Not fixed